src/distrib/make_installer.sh

```diff
@@ -84,4 +91,9 @@ install -m755 bin/* "\$PREFIX/bin"
 if ! [[ -e \$PREFIX/bin/opam ]]; then
   install_opam
 fi
+
+}
+
+# Now that we know the whole script has downloaded, run it.
+_ "$0" "$@"
```

Comment:
We're doing crazy meta-bash here:
```suggestion
_ "\$0" "\$@"
```

Reply:
Ahh, good point!